MobileCenter/MobileCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -45,56 +38,80 @@ - (BOOL)saveLog:(id<MSLog>)log withGroupID:(NSString *)groupID {
   }
   NSData *logData = [NSKeyedArchiver archivedDataWithRootObject:log];
   NSString *base64Data = [logData base64EncodedStringWithOptions:NSDataBase64EncodingEndLineWithLineFeed];
-  NSString *addLogQuery = [NSString stringWithFormat:@"insert or replace into %@ values ('%@', '%@')",
-                           kMSLogTableName, groupID, base64Data];
+  NSString *addLogQuery = [NSString
+      stringWithFormat:@"insert or replace into %@ values ('%@', '%@')", kMSLogTableName, groupID, base64Data];
   return [self.connection executeQuery:addLogQuery];
 }
 
 - (NSArray<MSLog> *)deleteLogsForGroupID:(NSString *)groupID {
-  NSArray<MSLog> *logs = [self getLogsWith:groupID];
-  [self deleteLogsWith:groupID];
+  NSArray<MSLog> *logs = [self getLogsWithGroupID:groupID];
+  [self deleteLogsWithGroupID:groupID];
   return logs;
 }
 
-- (void)deleteLogsForId:(NSString *)logsId withGroupID:(NSString *)groupID {
+- (void)deleteLogsForId:(NSString *)batchId withGroupID:(NSString *)groupID {
```

Comment:
Yes it's on purpose, handling this one will be another PR. I can silent the warning for now.